src/modules/sdk/objections/types.ts

```diff
@@ -19,6 +19,8 @@ export enum ObjectionStatus {
   OPEN = "OPEN",
   SUBMITTED = "SUBMITTED",
   PROCESSED = "PROCESSED",
+  INELIGIBLE_COMPANY_STRUCK_OFF = "INELIGIBLE_COMPANY_STRUCK_OFF",
```

Comment:
In java, these usually get highlighted when not in alphabetical order. I guess it doesn't matter in node, but it would be nice to have them in alphabetical order.....!